skimage/future/graph/rag.py

```diff
@@ -249,6 +249,45 @@ def copy(self):
         g.max_id = self.max_id
         return g
 
+    def fresh_copy(self):
+        """Return a fresh copy graph with the same data structure.
+
+        A fresh copy has no nodes, edges or graph attributes. It is
+        the same data structure as the current graph. This method is
+        typically used to create an empty version of the graph.
+        """
+        return RAG()
+
+    def subgraph(self, nbunch):
```

Comment:
I think you should be able to remove this method entirely.  The v1.x subgraph method and the v2.0 subgraph method should both work with the ```skimage/future/graph/``` code without error unless I'm missing something.